app/models/user.rb

```diff
@@ -63,10 +63,6 @@ def send_two_factor_authentication_code(_code)
     # the user in by calling this method. However, we don't want a code to be
     # automatically sent until the user has reached the TwoFactorAuthenticationController,
     # where we prompt them to select how they would like to receive the OTP code.
-    # Sending an OTP code is not a User responsibility. It belongs either in the
-    # controller, or in a dedicated class that the controller sends messages to.
-    # Based on the delivery method chosen by the user, the controller calls the
-    # appropriate background job to send the code, such as SmsSenderOtpJob.
```

Comment:
note that I removed this part of the comment because it is understandable without this part.

The comment was already getting out of date because there is currently no class named `SmsSenderOtpJob`